letsencrypt/config.json

```diff
@@ -15,8 +15,8 @@
   },
   "map": ["ssl:rw"],
   "options": {
-    "email": null,
-    "domains": [null],
+    "email": "hello@home-assistant.io",
+    "domains": ["home-assistant.io"],
```

Comment:
Let that on `null`, otherwise people hit the start button without change it

Reply:
This was done because of the discussion in #881. Its correct that the error message, when using null is hard to understand.
Therefore using this approach would provide the user with an easy understandable error message, making the setup easier.

Reply:
It also causes unexpected behaviour, which is weird as well.

My suggestion would be: Make it an empty string, detect that empty string in the startup and make a nice proper error message with instructions.

@pvizeli What do you think?

Reply:
I see what you mean. But the error im refering too is appearing when you are trying to save the config, not when you start the container. Maybe a change of the config.json is needed to allow saving the null setting?

Reply:
The issue is, if we migrate to UI options, the null is an indicator to become a `*`